### PR TITLE
Blog comments simple

### PR DESCRIPTION
**Pull request by erh**

### CODE REVIEW

File: blog-comments-simple/README.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+     <html>
+         <head>
+             <script src="https://s3.amazonaws.com/stitch-sdks/js/library/stable/stitch.min.js"></script>
+             <script>
+              const client = new stitch.StitchClient('mdbw17s1-poiib');
+              const db = client.service('mongodb', 'mongodb-atlas').db('blog');
+     
+              function displayCommentsOnLoad() {
+                  client.anonymousAuth().then(displayComments);
```
Comment (edaniels) [inline blog-comments-simple/README.md]: `anonymousAuth()` is deprecated by just `login()`. See Example Snippet in Getting Started

Comment (erh) [inline blog-comments-simple/README.md]: Done

**Comment by erh** [comment]
Seems this PR was updated?

**Comment by edaniels** [comment]
Hmm? It looks like you closed #13 and opened up this (#14)

**Comment by erh** [comment]
That was yesterday when I didn't realize how PRs worked with branches.

**Comment by edaniels** [comment]
👍 You're good to go

**Pull request merged**